tasks/html-css-popup.md

```diff
@@ -0,0 +1,47 @@
+[![MIT Licensed][icon-mit]][license]
+[![Awesome][icon-awesome]][awesome]
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+[![Telegram][icon-chat]][chat]
+
+# HTML & CSS practice: Google-style Popup
+
+The goal of this task is to practice coding HTML & CSS from mockup. You are expected to deepen your knowledge of HTML & SCC and discover new previously unknown features.
+
+Requirements:
+- no JavaScipt
+- no external libraries or frameworks
+- browser support: Chrome only (feel free to use latest features)
+
+The popup is expected to have three states depending on click on the Google App button (third element from left). After first click the popup appears. After second click it is hidden. When popup is open, click on "More" button adds 10 more icons and forces the content of the popup to scroll.
+Mobile view is irrelevant for this task.  
+Initial state: non pop-up visible:
+
+![popup-hidden](../img/popup-hidden.png)
+
+Active state:
+
+![popup-visible](../img/popup-visible.png)
+
+Scrollable state:
+
+![popup-scrollable](../img/popup-scrollable.png)
+
+All interactive elements should be marked as such.
+
+![popup-hover-state](../img/popup-hover.png)
+
```

Comment:
`All interactive elements should have :focus state.`
Add image with `:focus` state (underscored caption).

Reply:
nice, thanks